shopfloor/actions/schema.py

```diff
@@ -38,6 +60,12 @@ def move_line(self, with_packaging=False, with_picking=False):
             "package_dest": self._schema_dict_of(
                 self.package(with_packaging=with_packaging), required=False
             ),
+            "suggested_package_dest": self._schema_list_of(
```

Comment:
why do you need these?

Reply:
We need these when we process order with multiple products. We need to suggest the destination for the following product that go in the same package.